CPT331.Core/ObjectModel/Coordinate.cs

```diff
@@ -153,14 +168,12 @@ public static Coordinate FromValues(double latitude, double longitude)
 			return coordinate;
 		}
 
-        /// <summary>
-        /// Converts the coordinate values to an equivalent string representation.
-        /// </summary>
-        /// <returns>The string representation of the value of this instance.</returns>
+		/// <summary>
+		/// Converts the coordinate values to an equivalent string representation.
+		/// </summary>
+		/// <returns>The string representation of the value of this instance.</returns>
 		public override string ToString()
 		{
-            int x = 1;
-            x.ToString();
```

Comment:
Whoops.. I was trying to get to the built-in c# Meta Data the lazy way. I forgot to delete it afterwards.

/uppercut